plugins/ufm_log_analyzer_plugin/src/loganalyze/logs_extraction/directory_extractor.py

```diff
@@ -30,20 +30,23 @@ def extract_files(self, files_to_extract: List[str],
         if not os.path.exists(destination):
             os.makedirs(destination)
 
-        # Convert the list to a set for faster lookup
         files_to_extract = set(files_to_extract)
         directories_to_extract = set(directories_to_extract)
         found_files = set()
         not_found_files = set(files_to_extract)
+        _, logs_with_dirs = self._split_based_on_dir(files_to_extract)
```

Comment:
why _?
please rename to a meaningfull name

Reply:
Since it is not used.
This function returns 2 items, for this case one is not needed but for the other cases it is needed